YourToDo/templates/auth/loggedin.html

```diff
@@ -0,0 +1,9 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    <div class="container">
+        <h1>Hey {{ user.username }}, you logged in!<h1>
```

Comment:
We should change this message to be something else in my opinion. Maybe "Hey <username>, you're logged in!" or not have a loggedin.html and redirect to the homepage